meta-mender-demo/recipes-mender/hello-mender/hello-mender.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Mender test recipe for persistent data files"
+LICENSE = "Apache-2.0"
+
+inherit deploy
+
+do_compile() {
+    dd if=/dev/urandom of=persistent.dat bs=4k count=100
+    dd if=/dev/urandom of=hello-mender bs=4k count=100
```

Comment:
Maybe use something predictable so it's easier to see that the right content made it onto the `/data` partition?